04_environment/design.tex

```diff
@@ -0,0 +1,289 @@
+
+The inspiration for our concept of foraging comes from the stag hunt game. Thus, it introduces elements from it. Specifically, this approach results in a game with \textbf{no dominant strategy} and in which \textbf{an agent's choice can impact another agent's}.
+
+The process of stag hunt is that each agent selects a method of hunting without the information of the other agents. The social welfare (total payoff) is significantly reduced if their choices differ. The selection of stag suggests that the agent is going for higher \textbf{payoffs} whilst the selection of hare indicates that the agent is \textbf{risk aversion} as both agents are required to stag hunt to get any personal welfare (see Definition~\ref{def:Welfare}).
+
+\subsubsection{Modifications to the Original concepts} 
+A few changes have to be made to make this dilemma viable for our multi-agent system, in addition to adding realism. 
+
+\begin{enumerate}
+    \item The dependency was changed to be based on the input resources rather than the number of islands participating. This added a more realistic element to the dilemma as in reality a hunt’s return would be based on the amount of resources (different possible resources: people, food, water and materials) entered rather than the number of islands participating.
+    \item Instead of a specified return, probabilistic return was implemented which adds a randomness element to the game to prevent pattern recognition. These probabilities can be tied to the population, disasters, forecasting and number of animals hunted.  
+\end{enumerate}
+
+Furthermore, the dynamics of our agents and allowing for a more complex decision-making interaction between islands and the environment have been explored. It is worth mentioning that some of these design ideas were implemented in the coursework. These changes have been made to make foraging more volatile and unpredictable, adding to the complexity of the system, allowing the teams to further evaluate the agents’ performance on observing and to use learned knowledge to take action.
+
+\subsubsection{Tier System}
+
+The deer hunting and fishing is divided into tiers, which depend on the total amount of input resources. Tiers represent the number of fish that can be caught in a given day, and the “zero” tier represents the cost associated with getting to the hunting location. Beyond the zero tier, the cost of catching another deer or fish will decrease as it is easier to catch the second animal than the first one, since the agents have already arrived at the hunting location. Therefore, if the agents do not even enough to reach the first tier, then they wouldn't get any returns. It should be noted there is a limit to how many animals the agents can catch in a given day.
```

Comment:
omit the "the" at the start: Deer hunting and fishing *are* divided into tiers, dependent on the total amount of input resources.